src/patternfly/components/Table/table.scss

```diff
@@ -432,26 +432,23 @@
 // Compact table
 // ==================================================================
 .pf-c-table.pf-m-compact {
+  &:not(.pf-m-border-rows) {
```

Comment:
I think it would be good to show an example for this modifier

Reply:
added